src/components/landing-page/cd.landing-page.vue

```diff
@@ -124,146 +146,177 @@ class App extends Vue {
     isDisabled: false,
   };
 
-  fileExplorerConfig = {
-    isReadOnly: false,
-    hasFolders: true,
-  };
-
   async created() {
-    for (let i = 0; i < schemaPaths.length; i++) {
-      const path = schemaPaths[i].path;
-      const name = schemaPaths[i].name;
-      const { default: schema } = await import(
-        /* @vite-ignore */ `@/schemas/hydroshare/schema.json`
-      );
-      const { default: uischema } = await import(
-        /* @vite-ignore */
-        `@/schemas/hydroshare/uischema.json`
-      );
-      const { default: defaults } = await import(
-        /* @vite-ignore */
-        `@/schemas/hydroshare/defaults.json`
-      );
-
-      this.schemaCollection.push({
-        index: i,
-        name,
-        schema,
-        uischema,
-        defaults,
-      });
-    }
-    this.selectedSchema = 0; // Initial repository schema to render
-    this.data = { ...this.data, ...this.defaults };
+    const schema: SchemaDefinition = {
+      title: "EditableScientificDataset",
+      type: "object",
+      additionalProperties: true,
+      properties: {
+        name: {
+          type: "string",
+          default: null,
+          title: "Name or title",
+          description: "A text string with a descriptive name or title for the resource."
+        },
+        description: {
+          type: "string",
+          default: null,
+          title: "Description or abstract",
+          description: "A text string containing a description/abstract for the resource."
+        }
+      }
+    };
```

Comment:
Good, this is from:
https://github.com/CUAHSI/HydroShare-Cloud-Native-Data/blob/develop/schema/src/json_schemas/editable_scientific_dataset_json_schema.json

It might be good not to embed this schema into the landing-page component, but rather to keep the schema in a separate file like `src/schemas/hydroshare/edit_schema.json` or similar